common-lib/src/cmr/common/util.clj

```diff
@@ -947,3 +947,13 @@
   (let [index (.indexOf s substring)]
    (when (>= index 0)
     index))))
+
+(defn scrub-token
+  "Cut out the middle of the token, leaving 10 characters on each end.
+  For the token that contains less than 10 characters(like tokens in the test),
+  remove the last character."
+  [token]
+  (let [token-length (.length token)]
```

Comment:
I think `count` is preferred over Java interop.

Reply:
Done.